src/settings.ts

```diff
@@ -1996,6 +1996,7 @@ export class RemotelySaveSettingTab extends PluginSettingTab {
             undefined,
             undefined,
             [],
+            false,
```

Comment:
Wonder if this should be true; having first sync be failed doesn't make sense to me, might add confusion after people reset metadata. Definitely a nitpick though and we can see the experience later & decide

Reply:
As in making the metadata deletion add to the sync history?

Reply:
Ah sorry, I see now the parameter is the deletion history - does false make sense as a value here vs undefined? definitely a nitpick, I was thinking it would log the sync as failed but I didn't finish tracing the code.

Reply:
All good, I'll make it undefined as I just rushed it a bit to see if I could get it working. 

:)